App/menus.rb

```diff
@@ -0,0 +1,59 @@
+# class box that contains the definition of all the menus of the application
+class Menus
+  def initialize; end
+
+  def main__menu
+    "1. registrarse \n
+    2. ingresar"
+  end
+
+  def mattress_menu
+    "1. consultar dinero guardado \n
+    2. agregar dinero \n
+    3. retirar dinero \n
+    4. regresar al menu \n"
+  end
+
+  def user_menu
+    "1. consultar saldo disponible \n
+    2. consultar saldo total \n
+    3. recargar \n
```

Comment:
A que se refiere con recargar?

Reply:
#1 
```suggestion
    3. recargar \n
```
@FernandoMorantes  esto lo hice por que asi se le llama en nequi a poner saldo en la cuenta